manifests/kots-config.yaml

```
@@ -191,26 +191,28 @@ spec:
           hidden: true
           type: text
           default: *when_is_k8s_platform
-        - name: isK0sPlatform
+        - name: isEmbeddedCluster
           title: Embedded Cluster Platform
-          when: &when_is_k0s_platform >-
-            {{repl $result :=  (eq Distribution "k0s") -}}
+          # TODO: For the moment we're checking if it's embedded cluster if we can't obtain the k8s platform, but we should change this to a more reliable way when allowed by Replicated
+          when: &when_is_embedded_cluster >-
+            {{repl $result :=  (empty Distribution) -}}
```

can you change this condition to be compatible for the future?
```suggestion
            {{repl $result := or (empty Distribution) (eq Distribution "k0s") -}}
```

---

We don't have a last word from replicated about the way they're going to let us check if it's an embedded cluster or not 🤔 I have some doubts here, I added the TODO in line 196 for that